Bottom: 6f417890840696100bc0938d4558e9685fe3cdd6
Top:    86cdc9a1be9082ecc3f33dcfd676a4b7ed6ee749
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-14 00:40:34 +0100

Refresh of hack-to-make-pretty-translucen

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 3346c85..847da1b 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -2299,7 +2299,6 @@ hid_gtk_init ()
   ghid_hid.fill_circle              = ghid_fill_circle;
   ghid_hid.fill_polygon             = ghid_fill_polygon;
   ghid_hid.fill_pcb_polygon         = ghid_fill_pcb_polygon;
-  ghid_hid.thindraw_pcb_polygon     = ghid_thindraw_pcb_polygon;
   ghid_hid.fill_rect                = ghid_fill_rect;
 
   ghid_hid.calibrate                = ghid_calibrate;
